### PR TITLE
OSDOCS-4096: Bumping 1.24 to 1.25 throughout doc set

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4096

Link to docs preview:
https://51372--docspreview.netlify.app/openshift-enterprise/latest/welcome/index.html
But the changes are throughout; I'm not putting a link to every single section :smile_cat: 

QE review:
- [ ] QE has approved this change.

Additional information:
Please ignore the file with the "TODO" in it for now, during peer review

### CODE REVIEW

File: monitoring/configmap-reference-for-cluster-monitoring-operator.adoc [hunk 1]
```diff
@@ -587,3 +587,5 @@ component.
 |thanosRuler |link:#thanosrulerconfig[ThanosRulerConfig] |ThanosRuler
 defines configuration for the Thanos Ruler component
 |===
+
+// TODO: What to do about these 1.24 instances. This file is only in main and 4.10, can we delete it from main?
```
Comment (bergerhoffer) [inline monitoring/configmap-reference-for-cluster-monitoring-operator.adoc:591]: From this PR https://github.com/openshift/openshift-docs/pull/48344. Need to investigate

**Comment by bergerhoffer** [comment]
/label peer-review-needed

Please ignore the file marked with the "TODO" for now.

File: modules/restore-replace-stopped-baremetal-etcd-member.adoc [hunk 2]
```diff
@@ -434,7 +434,7 @@ spec:
 EOF
 ----
 +
-[Note] 
+[Note]
```
Comment (jldohmann) [inline modules/restore-replace-stopped-baremetal-etcd-member.adoc:437]: Kind of out of scope for this PR, but minor thing:
```suggestion
[NOTE]
```

**Review by jldohmann — APPROVED** [review]
LGTM, though I did leave a couple comments below

File: monitoring/configmap-reference-for-cluster-monitoring-operator.adoc (hunk 1 above)
Comment (bergerhoffer) [reply monitoring/configmap-reference-for-cluster-monitoring-operator.adoc:591]: Likely will be removed from main, so no changes needed w/ this PR

File: modules/dr-restoring-cluster-state.adoc [hunk 3]
```diff
@@ -183,7 +183,7 @@ $  ssh -i <ssh-key-path> core@<master-hostname>
 sh-4.4# pwd
 /var/lib/kubelet/pki
 sh-4.4# ls
-kubelet-client-2022-04-28-11-24-09.pem  kubelet-server-2022-04-28-11-24-15.pem
+kubelet-client-2022-04-28-11.25.09.pem  kubelet-server-2022-04-28-11-24-15.pem
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/dr-restoring-cluster-state.adoc:186]: Good catch! Not intentional, I'll revert, thanks!

File: modules/restore-replace-stopped-baremetal-etcd-member.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/restore-replace-stopped-baremetal-etcd-member.adoc:437]: I probably won't change it for this PR, but I'll try to catch it the next time I'm doing some random cleanup!

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please review these kube bumps? Thanks!

**Pull request merged**